### PR TITLE
Fix split v2 arbitrary allocations

### PR DESCRIPTION
**Pull request by r0ohafza**

### CODE REVIEW

File: packages/splits-sdk/src/client/splitV2.ts [hunk 1]
```diff
@@ -280,13 +273,19 @@ class SplitV2Transactions extends BaseTransactions {
...
+          ),
+          distributionIncentive: getNumberFromPercent(
+            split.distributorFeePercent,
+          ),
         },
         token,
         distributorAddress,
       ],
       transactionOverrides,
+      value: ZERO,
```
Comment (mihoward21) [inline packages/splits-sdk/src/client/splitV2.ts:288]: is this necessary? would have thought it's the default. fine to leave, just curious

File: packages/splits-sdk/src/subgraph/types.ts [hunk 2]
```diff
@@ -678,7 +678,7 @@ export type IWaterfallModule = IAccountSharedFields & {
 
 export type IHolder = {
   address: Address
-  ownership: number
+  ownership: bigint
```
Comment (mihoward21) [inline packages/splits-sdk/src/subgraph/types.ts:681]: feel pretty good we updated everywhere that uses this? and the irecipient?

**Review by mihoward21 — APPROVED** [review]
looks good. just double check those type changes, always a little nervous about that. but seems good

File: packages/splits-sdk/src/subgraph/types.ts (hunk 2 above)
Comment (r0ohafza) [reply packages/splits-sdk/src/subgraph/types.ts:681]: yeah, linter not throwing anything either

**Pull request merged**